package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "21.0.3",
+  "version": "21.0.12-upgrade-vuln-pkgs-beta",
```

Comment:
why 12-upgrade-vuln-pkgs-beta? what's between .3 and .12? why is this beta? are there more work to be done?

Reply:
We publish a beta to be reviewed and tested with hof services before publishing a new version, 12 was the next available for beta version